### PR TITLE
Add a readme

### PR DESCRIPTION
**Pull request by RomanTsukanov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!-- default badges list -->
+[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
+<!-- default badges end -->
+# DevExtreme DataGrid - How to modify the persisted state
+
+This example shows how to exclude component settings that you do not want restored after the page is reloaded.
```
Comment (NikolaevaIrina) [inline README.md]:
```suggestion
This example shows how to exclude items from component settings that are restored after the page is reloaded.
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+In this example, you can reorder grid columns, sort and group data, and perform other modifications in the UI. After you reload the page, all the modifications persist, except filters applied to the grid columns.
```
Comment (NikolaevaIrina) [inline README.md]:
```suggestion
In this example, you can reorder grid columns, sort and group data, and perform other modifications in the UI. After you reload the page, all the modifications persist, except filters applied to grid columns.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+The demonstrated approach can be used with the following UI components:
```
Comment (NikolaevaIrina) [inline README.md]:
```suggestion
You can use the demonstrated approach with the following UI components:
```

**Review by NikolaevaIrina — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
+<!-- default badges list -->
+[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
+<!-- default badges end -->
+# DevExtreme DataGrid - How to modify the persisted state
+
+This example shows how to exclude component settings that you do not want to be restored after the page is reloaded.
```
Comment (arminal) [inline README.md]:
```suggestion
This example shows how to exclude component settings that you don't need restored after the page is reloaded.
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+In this example, you can reorder grid columns, sort and group data, and perform other modifications in the UI. After you reload the page, all the modifications persist, except filters applied to grid columns.
```
Comment (arminal) [inline README.md]:
```suggestion
In this example, you can reorder grid columns, sort and group data, and perform other modifications in the UI. After you reload the page, all modifications will persist, except filters applied to grid columns.
```

**Comment by arminal** [comment]
#GrammarReview

Your sentence construction needs review. When I read what you wrote, I can almost tell you are translating. There is a writing sophistication that's just missing sometimes.

**Pull request merged**